lib/index.ts

```diff
@@ -276,10 +276,14 @@ export default class Backburner {
       }
     }
 
-    if (length === 1) {
-      return method();
-    } else if (length === 2) {
-      return method.call(target);
+    let onError = getOnError(this.options);
+
+    if (onError) {
+      try {
```

Comment:
^ and I think keeping `try...catch` in its own method is way to go to I can make another PR after merging this to add that